src/flat-button.jsx

```diff
@@ -232,17 +199,23 @@ const FlatButton = React.createClass({
     } = this.props;
 
     const {
-      buttonColor,
-      buttonHeight,
-      buttonMinWidth,
-      disabledTextColor,
-      buttonFilterColor,
-      primaryTextColor,
-      secondaryTextColor,
-      textColor,
-      textTransform,
-    } = this.constructor.getRelevantContextKeys(this.state.muiTheme);
-
+      button: {
+        color: buttonColor,
```

Comment:
Geez, I looked at the diff 100 times and for some reason I could not see this :rage:. Thanks! This fixed everything. :+1: